streamProcessor/videoStreamCollector.py

```diff
@@ -1,80 +1,89 @@
 import os
-import json
 import base64
 import logging
 import config
-
-from pyspark import SparkContext
-from pyspark.streaming import StreamingContext
-from pyspark.streaming.kafka import KafkaUtils
+from pyspark.sql.functions import from_json, col
+from pyspark.sql.types import *
 from pyspark.sql import SparkSession
 from resources import face_scrapper
 
 
-
 def init_spark():
-    sc = SparkContext(appName="videoStreamCollector")
-    ssc = StreamingContext(sc, config.Config.BATCH_DURATION)
-    sc.setLogLevel("WARN")
-    return sc, ssc
+    spark = SparkSession \
+        .builder \
+        .appName("videoStreamCollector") \
+        .master("master") \
```

Comment:
Is this the name of the spark master? If yes, it should be read from the environment variable specified in the Dockerfile.